packages/replay/src/canvas.ts

```diff
@@ -46,9 +59,32 @@ export class ReplayCanvas implements Integration {
       _experiments: {
         canvas: {
           ...this._canvasOptions,
-          manager: getCanvasManager,
+          manager: opts =>
```

Comment:
With the above suggestion, this can be 

```suggestion
          manager: () => this.canvasManager
```